src/product_pages/index.html

```diff
@@ -23,7 +23,6 @@ <h2 class="divider" id="productNameWithoutBrand"></h2>
         </div>
       </section>
     </main>
-    <script src="../js/checkout.js" type="module" defer></script>
     <script src="../js/product.js" type="module" defer></script>
```

Comment:
I've found that a lot of subtle bugs happen because scripts compete with each other and race conditions happen. I try to only have one entry point script. It's not mandatory, but it does make things a little easier.